packages/slate-editor/src/components/EditorBlock/EditorBlock.tsx

```diff
@@ -113,9 +113,7 @@ export const EditorBlock = forwardRef<HTMLDivElement, Props>(function (
                 [styles.void]: isVoid,
                 [styles.extendedHitArea]: extendedHitArea,
             })}
-            data-slate-type={element.type}
-            data-slate-value={JSON.stringify(element)}
```

Comment:
These two were moved to the `renderElement` wrapper callback. So these are automatically provided within `attributes` array to every rendered element.